libminifi/src/Configuration.cpp

```diff
@@ -179,9 +179,9 @@ std::vector<std::string> Configuration::getSensitiveProperties(const std::functi
 }
 
 bool Configuration::validatePropertyValue(const std::string& property_name, const std::string& property_value) {
-  for (const auto& config_property: Configuration::CONFIGURATION_PROPERTIES) {
-    if (config_property.name == property_name) {
-      return config_property.validator->validate(property_name, property_value).valid();
+  for (const auto& [config_property_name, config_property_validator]: Configuration::CONFIGURATION_PROPERTIES) {
+    if (config_property_name == property_name) {
+      return config_property_validator->validate(property_name, property_value).valid();
```

Comment:
Now that it's not a vector anymore, there is a more efficient way to do lookup in CONFIGURATION_PROPERTIES